wdl/MergeCohortVcfs.wdl

```diff
@@ -161,120 +104,11 @@ task MergeDepthVcfs {
 
   output {
     File merged_vcf = "~{prefix}.vcf.gz"
-    File regeno_merged_depth_clustered = "~{cohort}.regeno.merged_depth_clustered.bed"
   }
   command <<<
-
     set -euxo pipefail
     /opt/sv-pipeline/04_variant_resolution/scripts/merge_vcfs.py ~{write_lines(vcfs)} ~{prefix}.vcf
-    rm ~{sep=' ' vcfs}
```

Comment:
What happened to this `rm`?

Reply:
Since the rest of the task got moved to 04b, I didn't think it was necessary to remove the input VCFs anymore - it was there originally to save on disk space for the rest of the task